sdk/storage/storage-queue/package.json

```diff
@@ -41,7 +41,8 @@
     "test": "npm run clean && npm run build:test && npm run integration-test",
     "unit-test:browser": "cross-env TEST_MODE=playback npm run integration-test:browser",
     "unit-test:node": "cross-env TEST_MODE=playback npm run integration-test:node",
-    "unit-test": "npm run unit-test:node && npm run unit-test:browser"
+    "unit-test": "npm run unit-test:node && npm run unit-test:browser",
+    "emulator-tests": "cross-env TEST_MODE=emulator-tests && npm run test:node"
```

Comment:
`"emulator-tests"`
-  87/96 passing - 9 tests failed (with the emulator)

Reply:
what kind of failures?

Reply:
There maybe emulator issues, which kind of failures did you get?

Reply:
[ Errors are hidden in the dropdowns ]

 <details open>
<summary> 1) MessageIdClient
       update and delete empty message with default parameters:</summary>
<br>
<pre>
     AssertionError [ERR_ASSERTION]: undefined == true
      at path\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\nise\1.5.1\node_modules\nise\nise.js:4173:33
      at step (path\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:136:27)
      at Object.next (path\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:117:57)
      at fulfilled (path\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:107:62)      at process._tickCallback (internal/process/next_tick.js:68:7)
</pre>
</details>

 <details>
<summary>   2) MessagesClient
       enqueue, peek, dequeue and clear message with default parameters:</summary>
<br>
<pre>
     AssertionError [ERR_ASSERTION]: undefined == true
      at path\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\nise\1.5.1\node_modules\nise\nise.js:4175:48
      at step (path\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:136:27)
      at Object.next (path\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:117:57)
      at fulfilled (path\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:107:62)      at process._tickCallback (internal/process/next_tick.js:68:7)
</pre>
</details>

 <details>
<summary> 
  3) QueueClient
       getProperties with default/all parameters:</summary>
<br>
<pre>
     AssertionError [ERR_ASSERTION]: undefined == true
      at path\azure-sdk-for-js\sdk\storage\storage-queue\dist-test\index.node.js:12754:3588
      at step (path\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:136:27)
      at Object.next (path\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:117:57)
      at fulfilled (path\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:107:62)      at process._tickCallback (internal/process/next_tick.js:68:7)
</pre>
</details>

 <details>
<summary> 
  4) QueueServiceClient
       listQueues with default parameters:</summary>
<br>
<pre>
     AssertionError [ERR_ASSERTION]: undefined == true
      at path\azure-sdk-for-js\sdk\storage\storage-queue\dist-test\index.node.js:12760:1626
      at step (path\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:136:27)
      at Object.next (path\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:117:57)
      at fulfilled (path\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:107:62)      at process._tickCallback (internal/process/next_tick.js:68:7)
</pre>
</details>

 <details>
<summary> 
  5) QueueServiceClient
       getProperties with default/all parameters:</summary>
<br>
<pre>
     AssertionError [ERR_ASSERTION]: undefined == true
      at path\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\nise\1.5.1\node_modules\nise\nise.js:4187:68
      at step (path\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:136:27)
      at Object.next (path\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:117:57)
      at fulfilled (path\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:107:62)      at process._tickCallback (internal/process/next_tick.js:68:7)
</pre>
</details>

 <details>
<summary> 
  6) MessageIdClient Node.js only
       update message with 64KB characters including special char which is computed after encoding:</summary>
<br>
<pre>
     AssertionError [ERR_ASSERTION]: undefined == true
      at path\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\nise\1.5.1\node_modules\nise\nise.js:4214:62
      at step (path\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:136:27)
      at Object.next (path\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:117:57)
      at fulfilled (path\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:107:62)      at process._tickCallback (internal/process/next_tick.js:68:7)
</pre>
</details>

 <details>
<summary> 
  7) MessagesClient Node.js only
       enqueue, peek, dequeue with 64KB characters including special char which is computed after encoding:</summary>
<br>
<pre>
     Error: <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<Error>
  <Code>RequestBodyTooLarge</Code>
  <Message>The request body is too large and exceeds the maximum permissible limit.
RequestId:1f7b63f0-6723-4f42-9e25-1ee4eac82f95
Time:2019-09-20T07:06:39.940Z</Message>
  <MaxLimit>65536</MaxLimit>
</Error>
      at new RestError (path\azure-sdk-for-js\sdk\core\core-http\lib\restError.ts:18:5)
      at path\azure-sdk-for-js\sdk\core\core-http\lib\policies\deserializationPolicy.ts:117:27
      at process._tickCallback (internal/process/next_tick.js:68:7)
</pre>
</details>

 <details>
<summary> 
  8) QueueClient Node.js only
       getAccessPolicy:</summary>
<br>
<pre>
     AssertionError [ERR_ASSERTION]: undefined == true
      at path\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\nise\1.5.1\node_modules\nise\nise.js:4215:53
      at step (path\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:136:27)
      at Object.next (path\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:117:57)
      at fulfilled (path\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:107:62)      at process._tickCallback (internal/process/next_tick.js:68:7)
</pre>
</details>

 <details>
<summary> 
  9) Shared Access Signature (SAS) generation Node.js only
       generateQueueSASQueryParameters should work for queue with access policy:</summary>
<br>
<pre>
     AssertionError [ERR_ASSERTION]: undefined == true
      at path\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\nise\1.5.1\node_modules\nise\nise.js:4219:12
      at step (path\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:136:27)
      at Object.next (path\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:117:57)
      at fulfilled (path\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:107:62)      at process._tickCallback (internal/process/next_tick.js:68:7)
</pre>
</details>

Reply:
[ Errors are hidden in the dropdowns ]




 <details open>
<summary> 
  1) MessageIdClient
       update and delete empty message with default parameters:</summary>
<br>
<pre>
     AssertionError [ERR_ASSERTION]: undefined == true
      at C:\Users\harsh\Desktop\Work\azure-sdk-for-js\sdk\storage\storage-queue\dist-test\index.node.js:12749:8693
      at step (C:\Users\harsh\Desktop\Work\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:136:27)
      at Object.next (C:\Users\harsh\Desktop\Work\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:117:57)
      at fulfilled (C:\Users\harsh\Desktop\Work\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:107:62)
      at process._tickCallback (internal/process/next_tick.js:68:7)</pre>
</details>
 <details>
<summary> 
  2) MessagesClient
       enqueue, peek, dequeue and clear message with default parameters:</summary>
<br>
<pre>
     AssertionError [ERR_ASSERTION]: undefined == true
      at C:\Users\harsh\Desktop\Work\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\nise\1.5.1\node_modules\nise\nise.js:4181:4
      at step (C:\Users\harsh\Desktop\Work\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:136:27)
      at Object.next (C:\Users\harsh\Desktop\Work\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:117:57)
      at fulfilled (C:\Users\harsh\Desktop\Work\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:107:62)
      at process._tickCallback (internal/process/next_tick.js:68:7)</pre>
</details>
 <details>
<summary> 
  3) QueueClient
       getProperties with default/all parameters:</summary>
<br>
<pre>
     AssertionError [ERR_ASSERTION]: undefined == true
      at C:\Users\harsh\Desktop\Work\azure-sdk-for-js\sdk\storage\storage-queue\dist-test\index.node.js:12754:3588
      at step (C:\Users\harsh\Desktop\Work\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:136:27)
      at Object.next (C:\Users\harsh\Desktop\Work\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:117:57)
      at fulfilled (C:\Users\harsh\Desktop\Work\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:107:62)
      at process._tickCallback (internal/process/next_tick.js:68:7)</pre>
</details>
 <details>
<summary> 
  4) QueueServiceClient
       listQueues with default parameters:</summary>
<br>
<pre>
     AssertionError [ERR_ASSERTION]: undefined == true
      at C:\Users\harsh\Desktop\Work\azure-sdk-for-js\sdk\storage\storage-queue\dist-test\index.node.js:12760:1626
      at step (C:\Users\harsh\Desktop\Work\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:136:27)
      at Object.next (C:\Users\harsh\Desktop\Work\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:117:57)
      at fulfilled (C:\Users\harsh\Desktop\Work\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:107:62)
      at process._tickCallback (internal/process/next_tick.js:68:7)</pre>
</details>
 <details>
<summary> 
  5) QueueServiceClient
       getProperties with default/all parameters:</summary>
<br>
<pre>
     AssertionError [ERR_ASSERTION]: undefined == true
      at C:\Users\harsh\Desktop\Work\azure-sdk-for-js\sdk\storage\storage-queue\dist-test\index.node.js:12763:1799
      at step (C:\Users\harsh\Desktop\Work\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:136:27)
      at Object.next (C:\Users\harsh\Desktop\Work\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:117:57)
      at fulfilled (C:\Users\harsh\Desktop\Work\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:107:62)
      at process._tickCallback (internal/process/next_tick.js:68:7)</pre>
</details>
 <details>
<summary> 
  6) MessageIdClient Node.js only
       update message with 64KB characters including special char which is computed after encoding:</summary>
<br>
<pre>
     AssertionError [ERR_ASSERTION]: undefined == true
      at C:\Users\harsh\Desktop\Work\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\nise\1.5.1\node_modules\nise\nise.js:4220:33
      at step (C:\Users\harsh\Desktop\Work\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:136:27)
      at Object.next (C:\Users\harsh\Desktop\Work\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:117:57)
      at fulfilled (C:\Users\harsh\Desktop\Work\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:107:62)
      at process._tickCallback (internal/process/next_tick.js:68:7)</pre>
</details>
 <details>
<summary> 
  7) MessagesClient Node.js only
       enqueue, peek, dequeue with 64KB characters including special char which is computed after encoding:</summary>
<br>
<pre>
     Error: <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<Error>
  <Code>RequestBodyTooLarge</Code>
  <Message>The request body is too large and exceeds the maximum permissible limit.
RequestId:170b318b-06a2-41f4-bb6f-ad95c0206359
Time:2019-09-20T16:49:39.532Z</Message>
  <MaxLimit>65536</MaxLimit>
</Error>
      at new RestError (C:\Users\harsh\Desktop\Work\azure-sdk-for-js\sdk\core\core-http\lib\restError.ts:18:5)
      at C:\Users\harsh\Desktop\Work\azure-sdk-for-js\sdk\core\core-http\lib\policies\deserializationPolicy.ts:117:27
      at process._tickCallback (internal/process/next_tick.js:68:7)</pre>
</details>
 <details>
<summary> 
  8) QueueClient Node.js only
       getAccessPolicy:</summary>
<br>
<pre>
     AssertionError [ERR_ASSERTION]: undefined == true
      at C:\Users\harsh\Desktop\Work\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\nise\1.5.1\node_modules\nise\nise.js:4221:6
      at step (C:\Users\harsh\Desktop\Work\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:136:27)
      at Object.next (C:\Users\harsh\Desktop\Work\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:117:57)
      at fulfilled (C:\Users\harsh\Desktop\Work\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:107:62)
      at process._tickCallback (internal/process/next_tick.js:68:7)</pre>
</details>
 <details>
<summary> 
  9) Shared Access Signature (SAS) generation Node.js only
       generateQueueSASQueryParameters should work for queue with access policy:</summary>
<br>
<pre>
     AssertionError [ERR_ASSERTION]: undefined == true
      at C:\Users\harsh\Desktop\Work\azure-sdk-for-js\sdk\storage\storage-queue\dist-test\index.node.js:12795:2703
      at step (C:\Users\harsh\Desktop\Work\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:136:27)
      at Object.next (C:\Users\harsh\Desktop\Work\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:117:57)
      at fulfilled (C:\Users\harsh\Desktop\Work\azure-sdk-for-js\common\temp\node_modules\.registry.npmjs.org\tslib\1.10.0\node_modules\tslib\tslib.js:107:62)
      at process._tickCallback (internal/process/next_tick.js:68:7)</pre>
</details>